src/Infrastructure/ConfiguredCookie.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Infrastructure;
+
+use Symfony\Component\HttpFoundation\Cookie;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ */
+class ConfiguredCookie {
```

Comment:
CookieBuilder is probably a better name, since this itself is not a cookie

Reply:
Naming things... Thank you, just amended the commit.